### PR TITLE
4. Output file - void saveFile()

### LINKED ISSUE #20: 4. Output file
void saveFile()
with use of protected class member std::string fileName_

### PR DESCRIPTION
**Pull request by majkel84**
saving example string to file
no actuall score

### CODE REVIEW

File: bowling/main-ut.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        testing::InitGoogleTest(&argc, argv);
+        auto code = RUN_ALL_TESTS();
+        return code;
+    }
+    catch (...)
+    {
+        return 0;
+    }
+    return 0;
+}
```
Comment (arafalski) [inline bowling/main-ut.cpp]: Redundant...

**Review by arafalski — CHANGES_REQUESTED** [review]

**Review by arafalski — CHANGES_REQUESTED** [review]
Rebase/merge with anraf1001:homework
Solve conflicts

File: bowling/main-ut.cpp (hunk 1 above)
Comment (majkel84) [reply bowling/main-ut.cpp]: done

File: bowling/file.cpp [hunk 2]
```diff
@@ -1 +1,23 @@
 #include "file.hpp"
+
+#include <experimental/filesystem>
```
Comment (arafalski) [inline bowling/file.cpp:3]: I think that we can use `<filesystem>` rather than `<experimental/filesystem>`

Comment (majkel84) [reply bowling/file.cpp:3]: g++ udpated
include changed

File: bowling/main-ut.cpp (hunk 1 above)
Comment (arafalski) [reply bowling/main-ut.cpp]: It returned :D

**Review by arafalski — COMMENTED** [review]
Resolve conflicts

**Review by arafalski — APPROVED** [review]

**Pull request merged**